<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherBoard</class>
 <widget class="QWidget" name="WeatherBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather Board</string>
  </property>
  <widget class="QLCDNumber" name="lcdNum_Temp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="lTemperature">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPressure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum_Pressure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum_Altitude">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum_Humidity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum_UV">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum_Visible">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum_IR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="lAltitude">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHumidity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="lUV">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>UV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lVisible">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Visible</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IR</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlotTempHumi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlotAltitude">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlotPressure">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlotUVAmbient">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lTemperature_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHumidity_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="lAltitude_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPressure_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure</string>
   </property>
  </widget>
  <widget class="QLabel" name="lVisible_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Visible</string>
   </property>
  </widget>
  <widget class="QLabel" name="lUV_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>UV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIR_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>meter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>hPa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lux</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>WEATHER Station</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Reconnect Serial</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_exitButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>440</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
